src/pynxtools/dataconverter/helpers.py

```diff
@@ -577,32 +577,66 @@ def is_value_valid_element_of_enum(value, elist) -> Tuple[bool, list]:
 NUMPY_FLOAT_TYPES = (np.half, np.float16, np.single, np.double, np.longdouble)
 NUMPY_INT_TYPES = (np.short, np.intc, np.int_)
 NUMPY_UINT_TYPES = (np.ushort, np.uintc, np.uint)
-
+# np int for np version 1.26.0
+np_int = (
+    np.intc,
+    np.int_,
+    np.intp,
+    np.int8,
+    np.int16,
+    np.int32,
+    np.int64,
+    np.uint8,
+    np.uint16,
+    np.uint32,
+    np.uint64,
+    np.unsignedinteger,
+    np.signedinteger,
+)
+np_float = (np.float16, np.float32, np.float64, np.floating)
```

Comment:
@lukaspie @RubelMozumder 
Cant't you just use np.integer and np.floating here directly? I thought all the specific types are subtypes of this.
https://numpy.org/doc/stable/reference/arrays.scalars.html

Reply:
@rettigl, you are right, I was also thinking, there might be a inheritance interface among the type. I did not give that much focus on it. I also see there are some type that are not in the list of types in the figure. I am creating an issue on it, we can make a test in this case.